helm/happa/templates/configmap.yaml

```diff
@@ -12,6 +12,13 @@ data:
   audience: {{ .Values.Installation.V1.GiantSwarm.API.Address }}
   cp-audience: {{ .Values.Installation.V1.GiantSwarm.OIDC.IssuerAddress }}
   ingress-base-domain: {{ .Values.Installation.V1.Guest.Kubernetes.IngressController.BaseDomain }}
+  installation-name: {{ .Values.Installation.V1.Name }}
   aws-capabilities-json: '{{ .Values.Installation.V1.Provider.AWS.EC2.Instance.Capabilities | toJson }}'
   azure-capabilities-json: '{{ .Values.Installation.V1.Provider.Azure.VM.VmSize.Capabilities | toJson }}'
   default-request-timeout-seconds: '{{ .Values.Installation.V1.GiantSwarm.Happa.DefaultRequestTimeoutSeconds }}'
+  provider: {{ .Values.Installation.V1.Provider.Kind }}
+
+  {{ if .Values.Installation.V1.Infra.TestingEnvironment }}
+  # Enables real user monitoring (RUM)
+  enable-rum: 'TRUE'
+  {{ end }}
```

Comment:
After https://github.com/giantswarm/installations/pull/1493 is merged, this is going to change. But I don't want to make it too complicated for now.